### DISCUSSION TITLE
Timelines for waivers

### DISCUSSION THREAD

**Discussion by bohsask** [110yards/fantasy#195] [General]
Just wondering what the processing day for waivers will be every week, there's no note up at the moment on the waivers page.   Are we doing overnight on Tuesday as the standard processing time?

**Comment by bohsask** [comment]
Ha ah ok I see they ran last night.  Why do we do such a quick turnaround?

**Reply by mdryden** [reply]
Here's how the end of week stuff works:

1) There is a scheduled task which runs every day at 8AM central time and checks whether anything needs to be done.  I call this "end of day".

2) When end of day runs, if waivers are not active and it's been at least 36 hours since the end of the last game of the week, then end of week will run.  End of week enables waivers and calculates the matchup results.

3) The next time end of day runs, the waivers will be processed.

Practically speaking, this means if the last game of the week is on a Saturday, the results will be calculated on Monday morning and waivers will be processed Tuesday morning.

This week, with the Monday game, it will be a quick turnaround.  Results will calculate on Wednesday morning, and waivers will run on Thursday morning, leaving only Thursday for non-waiver pickups.
